packages/server/lib/hooks/hooks.ts

```diff
@@ -51,18 +51,19 @@ export const connectionCreationStartCapCheck = async ({
     }
 
     const scriptConfigs = await getSyncConfigsWithConnections(providerConfigKey, environmentId);
+    if (scriptConfigs.length <= 0) {
+        return false;
+    }
```

Comment:
not sure this check is useful. If empty the for loop below will immediately exit

Reply:
yeah indeed